packages/mdl-snackbar/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MDLComponent from 'mdl-base';
+import MDLSnackbarFoundation from './foundation';
+
+export {MDLSnackbarFoundation};
+
+export default class MDLSnackbar extends MDLComponent {
+  static buildDom() {
+    const {ROOT: CSS_ROOT, TEXT, ACTION} = MDLSnackbarFoundation.cssClasses;
+
+    const root = document.createElement('div');
+    root.classList.add(CSS_ROOT);
+    root.setAttribute('aria-live', 'assertive');
+    root.setAttribute('aria-atomic', 'true');
+    root.setAttribute('aria-hidden', 'true');
+    root.innerHTML = `
+      <div class="${TEXT}"></div>
+      <button type="button" class="${ACTION}"></button>
+    `;
+
+    return root;
+  }
+
+  static attachTo(root) {
+    return new MDLSnackbar(root);
+  }
+
+  show(data) {
+    this.foundation_.show(data);
+  }
+
+  getDefaultFoundation() {
+    const {TRANS_END_EVENT_NAME, TEXT_SELECTOR, ACTION_SELECTOR} = MDLSnackbarFoundation.strings;
+    const getText = () => this.root_.querySelector(TEXT_SELECTOR);
+    const getAction = () => this.root_.querySelector(ACTION_SELECTOR);
+    /* eslint brace-style: "off" */
+    return new MDLSnackbarFoundation({
+      addClass: className => this.root_.classList.add(className),
+      removeClass: className => this.root_.classList.remove(className),
+      hide: () => this.root_.setAttribute('aria-hidden', 'true'),
```

Comment:
hmm, my concern here is that general method names like `hide` and `unhide` won't make it apparent that the client should be updating the `aria-hidden` attribute. Perhaps symmetric methods like `setAriaHidden()` and `unsetAriaHidden` would be more semantic?